modules/nw-understanding-networking-managing-traffic-within.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * networking/understanding-networking.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-understanding-networking-managing-traffic-within_{context}"]
+= Managing traffic within the cluster
+
+Your applications need to communicate with each other inside the cluster. {product-title} provides two primary mechanisms for internal traffic: direct pod-to-pod communication for simple exchanges and robust service discovery for reliable connections.
+
+== Pod-to-pod communication
+
+Pods communicate directly using the unique IP addresses assigned by the _pod network_. A pod on one node can send traffic directly to a pod on another node without any network address translation (NAT). This direct communication model is efficient for services that need to exchange data quickly. Applications can simply target another pod’s IP address to establish a connection.
+
+== Service discovery with DNS
```

Comment:
All titles and section headings must have an anchor ID.

OCP docs guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings